src/index.ts

```diff
@@ -235,6 +236,14 @@ async function getBuildOptions(
     plugins.push(ReportPlugin(report, cwd, cjsDts))
   }
 
+  if (hasCSSInput(entry)) {
```

Comment:
```suggestion
  if (target) {
```

Let's always add lightning css plugin. But lazy-load `lightningcss` on `unplugin-lightningcss` side.